File: native/Marketplace/app/src/main/java/com/example/marketplace/constants/CarData.kt
package com.example.marketplace.constants

object CarData {
    val CarBrands = listOf("Toyota", "Honda", "Ford", "BMW", "Mercedes-Benz", "Audi", "Seat").sorted()

    val CarModels = mapOf(
        "Toyota" to listOf("Yaris", "Supra", "Corolla", "Camry", "Prius", "C-HR").sorted(),
        "Honda" to listOf("Accord", "Civic", "CR-V").sorted(),
        "Ford" to listOf("Focus", "Mondeo", "EcoSport", "Kuga", "Fiesta", "Ka").sorted(),
        "BMW" to listOf("1-series", "2-series", "3-series").sorted(),
        "Mercedes-Benz" to listOf("C-klass", "A-klass", "E-klass", "S-klass").sorted(),
        "Audi" to listOf("A1", "A2", "A3", "A4", "A5", "A6", "A7", "A8", "R8").sorted(),
        "Seat" to listOf("Exeo", "Leon", "Ibiza").sorted(),
        "" to listOf()
    )

    val Colours = listOf("Black", "White", "Red", "Blue", "Silver", "Green", "Yellow").sorted()

    val YearsList = (1980..2024).reversed().toList()
}

File: non-native/marketplace/lib/constants/car_data.dart
class CarData {
  static List<String> carBrands = [
    "Toyota",
    "Honda",
    "Ford",
    "BMW",
    "Mercedes-Benz",
    "Audi",
    "Seat"
  ]..sort();

  static Map<String, List<String>> carModels = {
    "Toyota": ["Yaris", "Supra", "Corolla", "Camry", "Prius", "C-HR"]..sort(),
    "Honda": ["Accord", "Civic", "CR-V"]..sort(),
    "Ford": ["Focus", "Mondeo", "EcoSport", "Kuga", "Fiesta", "Ka"]..sort(),
    "BMW": ["1-series", "2-series", "3-series"]..sort(),
    "Mercedes-Benz": ["C-klass", "A-klass", "E-klass", "S-klass"]..sort(),
    "Audi": ["A1", "A2", "A3", "A4", "A5", "A6", "A7", "A8", "R8"]..sort(),
    "Seat": ["Exeo", "Leon", "Ibiza"]..sort(),
    "": []
  };

  static List<String> colours = [
    "Black",
    "White",
    "Red",
    "Blue",
    "Silver",
    "Green",
    "Yellow"
  ]..sort();

  static List<int> yearsList = List<int>.generate(50, (index) => 2024 - index);
}
